specs/error_state/ErrorCodeStore.md

```diff
@@ -1,41 +1,30 @@
 # ErrorCodeStore state
 
 ## Procedure
-`ErrorCodeStore` is for two original code store related errors: `CodeStoreOutOfGas` and 
-`MaxCodeSizeExceeded`.
-this type of error only occurs when executing create(create,create2) op code or tx deploy 
-transaction(tx.to = null).
+`ErrorCodeStore` is for two original code store related errors: `CodeStoreOutOfGas` and `MaxCodeSizeExceeded`. This type of error only occurs when executing create(create,create2) op code or tx deploy transaction(tx.to = null).
 
 ### EVM behavior
-When handling a CREATE-kind transaction, Initial bytecode opcodes will run and current call context is created.
-The final bytecode opcodes to store for new contract is the `RETURN` opcode of init codes result.
+When handling a CREATE-kind transaction, Initial bytecode opcodes will run and current call context is created. The final bytecode opcodes to store for new contract is the `RETURN` opcode of init codes result.
 
-`RETURN` opcode returns memory [`offset`...`offset` + `length`] content as bytecodes to store into state db.
-for returned bytecodes, store them cost additional gas.   
+`RETURN` opcode returns memory [`offset`...`offset` + `length`] content as bytecode to store into state db. For returned bytecode, store them cost additional gas.   
 
-`let CODE_DEPOSIT_BYTE_COST = 200
-code_store_cost = CODE_DEPOSIT_BYTE_COST * bytecodes' length
-`  
+```
+let CODE_DEPOSIT_BYTE_COST = 200
+code_store_cost = CODE_DEPOSIT_BYTE_COST * len(bytecodes)
+``` 
 
-if `code_store_cost` > gas left, it is `CodeStoreOutOfGas` case.  
-if returned bytecodes' length > `MAXCODESIZE` allowed in evm, it is 
-`MaxCodeSizeExceeded` case.  
+- If `code_store_cost` > gas left, it is `CodeStoreOutOfGas` case.
+- If returned bytecode length > `MAXCODESIZE`, it is `MaxCodeSizeExceeded` case.  
 
-In circuit bus mapping side, check these two code store errors in [here](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/bus-mapping/src/circuit_input_builder/input_state_ref.rs#L1148&L1155)
-when executing opcode is `RETURN` and call context is creating(`call.is_create == true`) meanwhile.  
+In circuit bus mapping side, check these two code store errors in [here](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/bus-mapping/src/circuit_input_builder/input_state_ref.rs#L1148&L1155). When executing opcode is `RETURN` and call context is creating(`call.is_create == true`) meanwhile.  
```

Comment:
Not changed in this PR, but I am not sure what "meanwhile" refers to?

Reply:
I think it means `at the same time`